code/modules/reagents/chemistry/reagents/drink_cold.dm

```diff
@@ -177,3 +177,13 @@
 /datum/reagent/consumable/drink/cold/rewriter/on_mob_life(mob/living/M)
 	M.Jitter(5)
 	return ..()
+
+/datum/reagent/consumable/drink/cold/arnold_palmer
+    name = "Arnold Palmer"
+    id = "arnold_palmer"
+    description = "A wholesome mixture of lemonade and iced tea."
+    color = "#8B5427" // rgb: 139, 84, 39
+    drink_icon = "arnoldpalmer"
+    drink_name = "Arnold Palmer"
+    drink_desc = "A wholesome mixture of lemonade and iced tea...looks like somebody didn't stir this one very well."
+    taste_description = "sweet and fizzy"
```

Comment:
```suggestion
/datum/reagent/consumable/drink/cold/arnold_palmer
	name = "Arnold Palmer"
	id = "arnold_palmer"
	description = "A wholesome mixture of lemonade and iced tea."
	color = "#8B5427" // rgb: 139, 84, 39
	drink_icon = "arnoldpalmer"
	drink_name = "Arnold Palmer"
	drink_desc = "A wholesome mixture of lemonade and iced tea...looks like somebody didn't stir this one very well."
	taste_description = "sweet and fizzy"
```

Reply:
this one ended up being counted as outdated so if it's still spaces rather than tabs feel free to re-suggest it

Reply:
Ah, right I didn't include all the changes in both. Just a second.